### DISCUSSION TITLE
Server Directory listing - Dark Mode - Apache and Nginx

### DISCUSSION THREAD

**Discussion by gigamaster** [leokhoa/laragon#810] [Show and tell]
Server Directory Index 

Web server directory index customization with preview, pagination, download — h5ai.

![SDI h5ai screenshot](https://github.com/gigamaster/server-directory-index/raw/main/screen.png)

Running in USB Drive   
Disable calculation of folder size   
Edit `_h5ai/private/conf/options.json`

### Apache httpd 2.2/2.4

In the root directory's .htaccess file add:

```bash
DirectoryIndex  index.html  index.php  /_h5ai/public/index.php
```

In httpd.conf add:

```bash
<Directory /var/www/public/>
	DirectoryIndex index.html index.php /_h5ai/server/php/index.php
</Directory>
```

PDF.js requires Apache to serve <code>.mjs</code> modules files.   
Add the mime type to the <code>.htaccess</code> file in your root directory where index.html can be accessed:

```bash
<IfModule mod_mime.c>
  AddType text/javascript js mjs
</IfModule>
```

More about Apache and Nginx 
https://github.com/gigamaster/server-directory-index/wiki 

Download
https://github.com/gigamaster/server-directory-index

**Comment by rescenic** [comment]
Thank you very much for the customizations.
